modules/recon/domains-vulnerabilities/ghdb.py

```diff
@@ -47,7 +47,7 @@ def module_run(self, domains):
             base_query = 'site:%s' % (domain)
             for dork in dorks:
                 # build query based on alternate list
-                if isinstance(dork, basestring):
+                if isinstance(dork, str):
```

Comment:
Definitely take a look at this. The separation of bytes and unicode from a common base class definitely forced some logic changes in the core.